cypress/integration/email-notification-form-spec.js

```diff
@@ -0,0 +1,215 @@
+describe('Mi Voz, Mi Voto email notification form user flow', () => {
+  beforeEach(() => {
+    //cy.checkPageA11y('http://localhost:3000')
```

Comment:
What do you think about removing the commented out line 3?

Reply:
So, by uncommenting that line (and commenting out the cy.visit), cypress runs accessibility tests!
![Screen Shot 2022-02-11 at 10 15 12](https://user-images.githubusercontent.com/724355/153637865-78a67c90-8ce2-4fa6-9093-9d23bd535058.png)
![Screen Shot 2022-02-11 at 10 16 00](https://user-images.githubusercontent.com/724355/153637881-0a028b27-e508-407a-aeab-f99e8ebbabf8.png)

Reply:
`$Form elements must have labels` is in reference to the language radio buttons.
`$Page should have a level-one heading` is in reference to the fact that the title of the site is wrapped in a `<p>` tag, and it should be an `<h1>` 
`$All page content should be contained by landmarks` ...  Well I wasn't sure about this one, but upon clicking the link, it sends me [here](https://dequeuniversity.com/rules/axe/4.4/region?application=axeAPI).  It looks the outtermost `div` in our Header component should be changed to a semantic `<header>` instead.  At least that is my guess.

Reply:
@ivonne-hernandez Would you like me to make these changes on this branch, before the merge?

Reply:
Absolutely, please move forward with making those changes. I would just ensure that the styling isn't impacted as you refactor each of these tags.